apps/frontend/src/pages/profile/profile.tsx

```diff
@@ -11,9 +11,14 @@ const Profile = (): JSX.Element => {
 	return (
 		<PageLayout>
 			<div className={styles["profile-layout"]}>
-				<h1 className={styles["title"]}>Profile</h1>
-
-				<EditUserForm user={authenticatedUser as UserAuthResponseDto} />
+				<div className={styles["profile-info"]}>
+					<h1 className={styles["title"]}>Profile</h1>
+					<EditUserForm user={authenticatedUser as UserAuthResponseDto} />
+				</div>
+				<div className={styles["divider"]} />
+				{authenticatedUser?.id && (
```

Comment:
Let's cast this type, we can only get to this page when we have authenticatedUser